inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp

```diff
@@ -1711,126 +1711,6 @@ CNNLayer::Ptr NodeConverter<ngraph::op::v0::Interpolate>::createLayer(const std:
     THROW_IE_EXCEPTION << "Interpolate operation should be converted to Interp";
 }
 
-template <>
-CNNLayer::Ptr NodeConverter<ngraph::op::v4::Interpolate>::createLayer(const std::shared_ptr<ngraph::Node>& layer) const {
```

Comment:
@vgavrilo , you removed correct code. This function should exist. You must remove LayerCreator for Interpolate-1 and Interpolate-4 in the inference-engine/src/readers/ir_reader/ie_ir_parser.cpp.

Reply:
Fixed.